### PR TITLE
feat: View saved addresses in BNMO checkout

### PR DESCRIPTION
**Pull request by annacarey**
Updated the schema in Force to get access to the `addressConnection`. Edited the UI to display the list of addresses and populate them in the component's state when selected.

Screen Shot 2021-01-21 at 12 51 26 PM

https://artsyproduct.atlassian.net/browse/PURCHASE-2300

### CODE REVIEW

File: src/v2/Apps/Order/Routes/Shipping/index.tsx [hunk 1]
```diff
@@ -325,6 +443,7 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
   }
 
   render() {
+    console.log(this.state)
```
Comment (sepans) [inline src/v2/Apps/Order/Routes/Shipping/index.tsx:446]: 🙈

File: src/v2/Apps/Order/Routes/Shipping/index.tsx [hunk 2]
```diff
@@ -87,25 +99,59 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
...
+        ? this.props.order.requestedFulfillment.phoneNumber
+        : ""
+    }
   }
 
   get startingAddress() {
-    return {
-      ...emptyAddress,
-      country: this.props.order.lineItems.edges[0].node.artwork.shippingCountry,
+    const defaultAddress = this.defaultAddress
```
Comment (sepans) [inline src/v2/Apps/Order/Routes/Shipping/index.tsx:122]: suggestion:
What do you think about renaming these to `defaultSavedAddress` or something similar to make it a bit more describtive?

**Review by sepans — APPROVED** [review]
🥇 🚀 This looks great 💯  A couple of small comments and suggestions. I will pull this PR to do some additional QA before merging it.

File: src/v2/Apps/Order/Routes/Shipping/index.tsx (hunk 1 above)
Comment (annacarey) [reply src/v2/Apps/Order/Routes/Shipping/index.tsx:446]: OHHHH whoops will delete!

File: src/v2/Apps/Order/Routes/Shipping/index.tsx (hunk 2 above)
Comment (annacarey) [reply src/v2/Apps/Order/Routes/Shipping/index.tsx:122]: yeah i'm down for this! Great suggestion. Thank you for your comments and ALL YOUR HELP on this @sepans

File: package.json [hunk 3]
```diff
@@ -100,6 +100,7 @@
     "bem-cn-lite": "3.0.0",
     "blueimp-file-upload": "9.22.1",
     "body-parser": "1.18.3",
+    "caniuse-lite": "^1.0.30001165",
```
Comment (sepans) [inline package.json:103]: Sorry I thought I commented on 👆🏽. is this needed? (also the `.ruby-version` file)

Comment (annacarey) [reply package.json]: thanks for catchint that! removed this change and the `.ruby-version` file but do I also need to remove the changes in the `yarn.lock` file too?

Comment (sepans) [reply package.json]: Yes I believe so. Or you can use yarn to remove it. I am guessing a `yarn add caniuse-lite` was accidentally run in the force directory, adding the library to `package.json` and `yarn.lock` so `yarn remove caniuse-lite` would remove it.

**Comment by sepans** [comment]
I did some more QA and it works great! 🎉 🥇 

I found a small problem with populating the country name when 2 saved addresses have different countries but definitely doesn't need to be fixed in this PR (I just wrote it here as a note for myself to check it when working on the Add/Edit address part)

**Comment by sepans** [comment]
Thanks again 🙏🏽

**Pull request merged**